module/apmawssdkgo/session_test.go

```diff
@@ -171,3 +174,31 @@ func TestDynamoDB(t *testing.T) {
 	}
 
 }
+
+func TestUnsupportedServices(t *testing.T) {
+	region := "us-west-2"
+	cfg := aws.NewConfig().
+		WithRegion(region).
+		WithDisableSSL(true).
+		WithCredentials(credentials.AnonymousCredentials)
+
+	session := session.Must(session.NewSession(cfg))
+	wrapped := WrapSession(session)
+	svc := athena.New(wrapped)
+
+	tx := apm.DefaultTracer.StartTransaction("send-email", "test-tx")
+	span := tx.StartSpan("test-span", "send-email", nil)
+	defer span.End()
+
+	ctx := apm.ContextWithSpan(context.Background(), span)
+	namedQuery := &athena.BatchGetNamedQueryInput{
+		NamedQueryIds: []*string{aws.String("query")},
+	}
+	// Setting a timeout so the request fails fast. This doesn't affect
+	// testing that we noop on unsupported services.
+	ctx, cancel := context.WithTimeout(ctx, time.Millisecond)
```

Comment:
each `BatchGetNamedQueryWithContext` was taking 7 seconds, so I added the timeout to speed that along.